aten/src/ATen/native/nested/NestedTensorBinaryOps.cpp

```diff
@@ -207,5 +242,16 @@ Tensor& fill_nested_(Tensor& self, const Tensor& value) {
   return self;
 }
 
+void _nested_op_dense_esuhm_cpu(Tensor& result, const Tensor& self, const Tensor& other, const NESTED_DENSE_OP& op) {
+  // TODO: implement CPU kernel
+  TORCH_CHECK(false);
+}
+
+REGISTER_ARCH_DISPATCH(nested_dense_elementwise_stub, DEFAULT, &_nested_op_dense_esuhm_cpu);
+REGISTER_AVX512_DISPATCH(nested_dense_elementwise_stub, &_nested_op_dense_esuhm_cpu);
+REGISTER_AVX2_DISPATCH(nested_dense_elementwise_stub, &_nested_op_dense_esuhm_cpu);
```

Comment:
nit: We don't need this for now. We can keep the CPU code in this file.

Reply:
I wasn't able to get it to compile when I only registered the cuda dispatch

Reply:
You can try `REGISTER_NO_CPU_DISPATCH `https://github.com/pytorch/pytorch/blob/6faa6c68e8b76fb68f3a2b2783685102d0e87c00/aten/src/ATen/native/DispatchStub.h#L265